contracts/cw-vest/src/msg.rs

```diff
@@ -0,0 +1,29 @@
+use crate::state::{CheckedPayment, Payment};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InstantiateMsg {
+    pub payments: Vec<Payment>,
+    pub admin_address: String,
```

Comment:
IMO admin should be optional. They should have the right to modify and cancel payments... if you don't want that, you don't want an admin.